build_tools/github/upload_anaconda.sh

```diff
@@ -3,6 +3,14 @@
 set -e
 set -x
 
+# Install Python 3.8 because of a bug with Python 3.9
+export PATH=$CONDA/bin:$PATH
+conda create -n upload -y python=3.8
+source activate upload
+conda install -y anaconda-client
+which anaconda
+
+# TODO: how to detect nightly builds for ARM on Circle CI?
```

Comment:
Maybe we can use `"$GITHUB_EVENT_NAME" == "schedule" or "$CIRCLE_BRANCH" == "main"`?

Reply:
I was thinking on something like this. We might want to design a uniform strategy that would work for any CI. Let's keep the nightly build config for a follow-up PR.

Reply:
+1.